web/api.go

```diff
@@ -45,18 +59,27 @@ func handleAPIError(w http.ResponseWriter, err error, logger golog.Logger, extra
 		temp := fmt.Sprintf("err not able to be converted to json (%s) (%s)", data, err)
 		w.WriteHeader(http.StatusInternalServerError)
 		_, err = w.Write([]byte(temp))
+
+		logger.Errorf("Internal API Issue: %s", err)
```

Comment:
can you use Errorw on both instead?

Reply:
Why do you prefer Errorw()? One downside i see a lot in our prod logs is that most errors only show in Stackdriver by default with the generic error like "Internal API Issue". This makes quickly assessing the actual error require an extra step of drilling into the error field.

With this the message includes both the generic error text and underlying error.

I'm also not opposed to `logger.Errorsw(fmt.Sprintf("Internal API Issue: %s", err.Error()), "error", err)` but feels a bit heavy handed.

Reply:
oh okay i assumed gcp would ultimately ingest structured logs

Reply:
when i used splunk this just worked out of the box